gspread/worksheet.py

```diff
@@ -985,6 +989,21 @@ def get(self, range_name=None, **kwargs):
 
         response = self.spreadsheet.values_get(range_name, params=params)
 
+        values = response.get("values", [])
+
+        if kwargs["maintain_size"] is True and is_full_a1_notation(range_name):
+            a1_range = get_a1_from_absolute_range(range_name)
```

Comment:
I feel like we overwrite the `range_name` above, or we could simply take the range_name and construct the complete range name for the GET request, then here we could simply do a call to `a1_range_to_grid_range` instead of de-constructing what we just built above.

it's no improvement it's just simpler to me, but it's up to you.

Reply:
The user provided `range_name` could be an absolute range name, so we have to parse it with `get_a1_from_absolute_range`.